src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -40,6 +42,11 @@ export const ConversationSpace: React.FC<ConversationSpaceProps> = ({
 
   const { recentVenueUsers } = useRecentVenueUsers({ venueName: venue?.name });
 
+  const {
+    isShown: showAvailableTables,
```

Comment:
same comment about the property hame; maybe `showOnlyAvailableTables` is better?
`toggleTablesVisibility` ?

Reply:
`toggleTablesVisibility` is fine for me

Reply:
fixed in 172c71750e0f23a6675546856b40288b3521892a